### DISCUSSION TITLE
python win32com <unknown>.Text

### DISCUSSION THREAD

**Discussion by liujiayi0042** [mhammond/pywin32#2548] [General]
Note that issues in this repository are only for bugs or feature requests in the pywin32.

I used win32com for PowerPoint, but 'python win32com .text' came up

* python==3.6andpywin32==225

**Comment by mhammond** [comment]
I've no idea what this report means - but it sounds like you have a support request for using powerpoint with pywin32 - see https://github.com/mhammond/pywin32#support for how to obtain support.
